modern/.env

```diff
@@ -1,5 +1,6 @@
 CONAN_VERSION=2.0.17
 DOCKER_TAG=2.0.17
+DOCKERHUB_USERNAME=conanio
```

Comment:
Is this `conanio` or `docker.io/conanio`? Sorry, I am still trying to understand 😅

Reply:
Yes, by default it uses docker.io in case passing nothing. It's because when using Artifactory, we actually pass the `center-c3i-docker.jfrog.io`.

Reply:
After merging this PR, I should create a new one to remove this switch between the jenkins agent version. I was a workaround, because in the past we didn't have a way to identify the jenkins agent version, but now we use the image tag.